packages/path_provider/path_provider/README.md

```diff
@@ -2,28 +2,206 @@
 
 [![pub package](https://img.shields.io/pub/v/path_provider.svg)](https://pub.dev/packages/path_provider)
 
-A Flutter plugin for finding commonly used locations on the filesystem. Supports Android, iOS, Linux, macOS and Windows.
+A Flutter plugin for finding commonly used locations on the filesystem. 
+Supports Android, iOS, Linux, macOS and Windows.
 Not all methods are supported on all platforms.
 
 ## Usage
 
 To use this plugin, add `path_provider` as a [dependency in your pubspec.yaml file](https://flutter.dev/docs/development/platform-integration/platform-channels).
 
-### Example
+Add the following import:
+```dart
+import 'package:path_provider/path_provider.dart';
+```
+
+## Supported platforms and paths
+
+Methods support by platform:
 
+| Dir | Android | iOS | Linux | macOS | Windows |
+| :--- | :---: | :---: | :---: | :---: | :---: |
+| Temporary | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| Application Support | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| Application Library | ❌️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| Application Documents | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ |
+| External Storage | ✔️ | ❌ | ❌ | ❌️ | ❌️ |
+| External Cache Directories | ✔️ | ❌ | ❌ | ❌️ | ❌️ |
+| External Storage Directories | ✔️ | ❌ | ❌ | ❌️ | ❌️ |
+| Downloads | ❌ | ❌ | ✔️ | ✔️ | ✔️ |
+
+### Temporary Directory
+Path to the temporary directory on the device that is suitable for storing caches of downloaded 
+files.
```

Comment:
I don't think we want to duplicate API docs in the README; that is asking for maintenance issues.